models/src/main/scala/coop/rchain/models/ParMap.scala

```diff
@@ -30,7 +30,7 @@ object ParMap {
       locallyFree: Coeval[BitSet],
       remainder: Option[Var]
   ) =
-    new ParMap(SortedParMap(seq), connectiveUsed, locallyFree.memoize, remainder)
+    new ParMap(new SortedParMap(seq.toMap), connectiveUsed, locallyFree.memoize, remainder)
```

Comment:
Oh, I thought you'd add the `apply()` method(s). It's OK, but please fix the Pretty instance for ParMap so that it still outputs compiling code (add `new `).

I'm sorry that I didn't leave a test that would tell you that earlier - it'd require compiling Pretty's output (I'd like to do it sometime in the future, simply had no time)

Reply:
Sorry, added `apply` methods now.